installer/charts/rhtap-infrastructure/templates/odf/post-install-hooks-rbac.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if .Values.odf.enabled }}
```

Comment:
If those service accounts and associated resources are useless once the deployment is complete, please add the [required annotations](https://github.com/redhat-appstudio/rhtap-cli/blob/529d4203f10b1230cb4f699c705e392a06786e0d/installer/charts/_common/_service-account.tpl#L7) so they get cleaned up.